com.somaxa8.earx.yml

```diff
@@ -0,0 +1,36 @@
+app-id: com.somaxa8.earx
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: earx
+finish-args:
+  - --system-talk-name=org.bluez
+  - --socket=wayland
+  - --device=all
+  - --allow=bluetooth
+
+modules:
+  - name: earx
+    build-options:
+      append-path: /usr/lib/sdk/rust-stable/bin
+    buildsystem: simple
+    build-commands:
+      - . /usr/lib/sdk/rust-stable/enable.sh
+      - mkdir .cargo
+      - cargo vendor > .cargo/config
+      - cargo fetch --verbose
```

Comment:
I am pretty the build will fail on this: cargo must be run in offline mode and in that case, either the dependencies vendored in the tarball, or generated with the builder tools:
https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo